### PR TITLE
Implemented endpoint for notification settings

### PR DESCRIPTION
**Pull request by NecyNazy**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -59,6 +63,37 @@
 			<artifactId>lombok</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<!-- RabbitMQ -->
```
Comment (Fifetoyi) [inline pom.xml:66]: remove comments

File: src/main/java/hng_java_boilerplate/notificationSettings/service/NotificationConsumerService.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.stereotype.Service;
+
+import java.sql.SQLOutput;
+
+@Service
+public class NotificationConsumerService {
+@RabbitListener(queues = RabbitMQConfig.QUEUE_NAME)
+    public void receiveNotification(String message){
+        System.out.println("Received message:" + message);
```
Comment (Fifetoyi) [inline src/main/java/hng_java_boilerplate/notificationSettings/service/NotificationConsumerService.java:14]: How would the frontend receive this message?

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1,7 +1,5 @@
-spring.application.name=hng-java-boilerplate
-
 # datasource configuration
-spring.datasource.url=jdbc:{connection_string}
+spring.datasource.url=jdbcgitg:{connection_string}
```
Comment (Fifetoyi) [inline src/main/resources/application.properties:2]: Why was this change made? Revert

File: src/main/resources/db/migration/V1__init_database.sql [hunk 4]
```diff
@@ -39,4 +39,13 @@ CREATE TABLE user_organisation (
     PRIMARY KEY (user_id, organisation_id),
     FOREIGN KEY (user_id) REFERENCES users(id),
     FOREIGN KEY (organisation_id) REFERENCES organisations(id)
+);
+
+-- V1__Create_notification_settings_table.sql
+CREATE TABLE notification_settings (
```
Comment (Fifetoyi) [inline src/main/resources/db/migration/V1__init_database.sql:45]: Table should be created in a flyway update schema with a unique version number

File: pom.xml (hunk 1 above)
Comment (NecyNazy) [reply pom.xml:66]: Okay

**Pull request closed without merging**